src/app/launcher/create-app/project-summary-createapp-step/project-summary-createapp-step.component.ts

```diff
@@ -129,6 +130,9 @@ export class ProjectSummaryCreateappStepComponent extends LauncherStep implement
           this.navToNextStep();
         }
       }, (error) => {
+        if (error) {
+          this.displaySetUpErrorResponse(error);
```

Comment:
error comes as a object I guess, generally? Is it not?

Reply:
Because it is directly accessed in html, that's why little curious

Reply:
the assumption here is string i guess

Reply:
yes that's the problem error comes as string, status and message all in same string. Even i was expecting it to be string.